ClinGen/clincoded
# Connects to #766. Connects to #905. Hook VCI up with dashboard and history

#### Connects to #766:

Interpretations with at least one valuation on it will be shown on the Dashboard under 'My Variant Interpretations'

![image](https://cloud.githubusercontent.com/assets/11320314/18219805/7c058a42-7121-11e6-97e7-9774a851227e.png)

Testing:

1. Get to VCI.
2. Check there is no entry for it under 'My Interpretations' on Dashboard
3. Create interpretation for variant
4. Check there is no entry for it under 'My interpretations' on Dashboard
5. Save evaluation for variant
6. Confirm that there is a new entry for variant under 'My Interpretations' on Dashboard
7. Associate disease to variant
8. Confirm that the entry now shows disease information on Dashboard

#### Connects to #905:

Interpretation/evaluation-related actions are now hooked up to the `curatorHistory` functionality from GCI. The involved schema changes to `curatorHistory` do not require an upgrade script as it adds a non-required sub-object, and it is not an edit of a previously-existing object.

Testing:

1. Add variant via VCI. Confirm that the variant adding event is recorded via Dashboard (Recent history).
2. Add interpretation. Confirm that the activity is NOT recorded via Dashboard (hidden log created).
3. Save evaluation(s). Confirm that the activity is recorded via Dashboard.
4. Associate disease to interpretation. Confirm that the activity is recorded via Dashboard.
5. Change/Save new evaluation(s). Confirm that the activity is recorded via Dashboard.

![image](https://cloud.githubusercontent.com/assets/4326866/17749011/ddb91834-6470-11e6-811b-e29735c55955.png)

#### Other change:

* Loading indicators from #925 is now utilized on Dashboard. Confirm that these work properly.
